.github/workflows/integration-test-macOS.yaml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   check:
-    runs-on: macos-10.15
```

Comment:
[Sounds like](https://github.com/ably/ably-cocoa/pull/1543#discussion_r1035924973) we're going to use `macos-latest`, so I think we need to update it to say, instead of "macOS 10.15", "the version of macOS specified by [GitHub Actions’ `macos-latest` runner image](https://github.com/actions/runner-images#available-images)".